simul/platform/platform.go

```diff
@@ -164,7 +166,7 @@ func ReadRunFile(p Platform, filename string) []*RunConfig {
 		// checking if format is good
 		vals := strings.Split(text, "=")
 		if len(vals) != 2 {
-			log.Fatal("Simulation file:", filename, " is not properly formatted ( key = value )")
+			log.Fatal("Simulation file:", filename, "line", line, "is not properly formatted ( key = value )")
```

Comment:
We miss some spaces here. I would do something like: 
`log.Fatal("Simulation file: ", filename, ":", line, " is not properly formatted ( key = value )")`

Reply:
log.Fatal uses the same formatting rules as fmt.Println, i.e. it adds spaces for every comma.